PlagiarismSubmissionSubmitListener.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace APP\plugins\generic\plagiarism;
+
+use PKP\observers\events\SubmissionSubmitted;
+use Illuminate\Events\Dispatcher;
+
+class PlagiarismSubmissionSubmitListener
+{
+
+    /** @var PlagiarismPlugin $plugin */
+    private $plugin;
+
+    public function __construct(PlagiarismPlugin $plugin)
+    {
+        $this->plugin = $plugin;
+    }
+
+    /**
+     * Maps methods with correspondent events to listen
+     */
+    public function subscribe(Dispatcher $events): void
+    {
+        $events->listen(
+            SubmissionSubmitted::class,
+            PlagiarismSubmissionSubmitListener::class

```

Comment:
instead of specifying the exact class name here, we can do it like 
```php
$events->listen(
            SubmissionSubmitted::class,
            [static::class, 'handle']
)
```